packages/expo-camera/ios/Next/VisionScannerDelegate.swift

```diff
@@ -0,0 +1,26 @@
+import UIKit
```

Comment:
I don't see any symbol that would require importing `UIKit`. Let's try to avoid importing it when possible – it would make things easier when we start supporting non-UIKit platforms (macOS)